dockerfile-latest

```diff
@@ -0,0 +1,41 @@
+FROM airsharedcontainers.azurecr.io/vstsdropdownloadcore:master AS app
+FROM airsharedcontainers.azurecr.io/msftkube AS msftkube
+FROM microsoft/vsts-agent:ubuntu-16.04-docker-17.12.0-ce-standard
+
+# Get vstsdropdownloadcore's built drop downloader application
+copy --from=app /app/* /app/
+copy --from=msftkube /msftkube/src/dist/*.wh /msftkube/package.wh
+
+RUN apt-get -qq update
+
+# Install Python2.7 and Python2.7 dependencies
```

Comment:
why do we need python 2.7 again? can't we get away from it?

Reply:
AZ CLI